wp-content/themes/ipbs-largo/less/widgets.less

```diff
@@ -9,4 +9,57 @@
         margin-bottom: @measure * 2;
         color: @blue;
     }
+}
+
+// member stations widget
+.widget.ipbs-member-stations {
+    margin-bottom: @measure * 2;
+    ul {
+        @media (min-width: 1000px) {
+            display: grid;
+            grid-template-columns: repeat(3, 1fr);
+        }
+        li {
+            display: flex;
+            align-items: center;
+            border-bottom: 1px solid @grey;
+            padding-top: @measure;
+            padding-bottom: @measure;
+            padding-left: @measure * 2;
```

Comment:
Can we have padding-right as well?

![Screen Shot 2020-01-30 at 14 16 38](https://user-images.githubusercontent.com/1754187/73482135-2801f500-436b-11ea-968b-aef1ab71e013.png)

Reply:
@benlk If we add a right padding it'll cause the same issues with the arrow that we were seeing in #68

Reply:
Here's a solution to the arrow problems, where the width of the element with the `.arrow` padding was moving around:

```diff
diff --git a/wp-content/themes/ipbs-largo/less/mixins.less b/wp-content/themes/ipbs-largo/less/mixins.less
index 477e1a7..edab44f 100644
--- a/wp-content/themes/ipbs-largo/less/mixins.less
+++ b/wp-content/themes/ipbs-largo/less/mixins.less
@@ -21,14 +21,19 @@
 }
 
 .arrow {
+    position: relative;
+    padding-right: 0.5rem;
     &::after {
         content: "\f345";
+        width: 1ch;
+        position: relative;
         font-family: 'dashicons';
         margin-left: 0.5rem;
         vertical-align: bottom;
-        transition: margin-left 0.3s;
+        transition: left 0.3s;
+        left: 0;
     }
     &:hover::after {
-        margin-left: 1rem;
+        left: 1ch;
     }
-}
\ No newline at end of file
+}

```

With element bounding boxes shown via pesticide.io:

![hover](https://user-images.githubusercontent.com/1754187/73484373-2fc39880-436f-11ea-97fb-dd9f52968d58.gif)

Reply:
This worked for the most part, except I had to change 

```
&:hover::after {
   left: 1ch;
}
```

to

```
&:hover::after {
&:hover::after {
   left: 10px;
}
```

Updated in https://github.com/INN/umbrella-ipbs/pull/73/commits/1f8ae0a5ef262595251080374e4a1f9423c5c2b9

Reply:
It even works with `float: right;`! Good work 🤙